### ISSUE TITLE
Home page down

### ISSUE THREAD

**Issue by trsh** [qiciengine/qiciengine#18]
http://www.qiciengine.com/ is down! Projects also seems very still. How are things going?

**Comment by cursedcoder** [comment]
It's not 100% uptime as for me, but it works now.

Project seems to be abandoned as far as I can see:
- no replies on email/forums
- registration doesn't work on forum
- no real versions last half a year

Maybe there are people who want to maintain fork?

**Comment by trsh** [comment]
Pity :(

**Comment by triptych** [comment]
I've messaged on Twitter, Facebook, and HTML5gamedevs.com and no response. However they did just release a major game on iOS so perhaps they've just been head's down coding? I wish someone from the core team would respond.

**Comment by cursedcoder** [comment]
@triptych they've responded on html5gamedevs www.html5gamedevs.com/topic/26264-qici-engine-dead/

**Comment by trsh** [comment]
Funny that guy who wrote it's dead, is a living qiciengine advertisement. If it is, for god sake write it down in repo, remove avatars and links from accounts, etc.

**Comment by Morabaraba** [comment]
@trsh It ruined my day when I found the html5gamedev thread, and I shared your sentiment. But after thinking about it I realised:

QICI Engine is still a full feature game engine. The core is open source, and have wonderful API and Manual Documentation, Google also indexed their GitBooks nicely so want to know how something work? "qici node", "qici tween", etc. There also a extensive list of demos!

Recent merge requests I made, also got accepted.

So the flood gates might not be streaming with updates; and yes the core team did go mia, most probably doing things like making a living.

Personally I'm a bit disgusted in the github and open source community. Since this project have been open sourced the community has just been a bag of needs. Taking strides like "Let't rename this", "Here is how we believe you must structure your code", etc.

I might be speaking out of line here. But I also believe the team hoped the community will pick it up faster. But even with a intuitive engine like QICI, game dev'ing stays hard, takes long, and is a niche field.

Finally, I'm releasing a game soon to Chrome Apps and Android made with QICI. I will be making the source avail, and have also been documenting the process in short 5 min video series I want to release with it. Hence why this issue ruined my day.

But I'm hopeful.

**tl;dr;** I'm going to continue to use QICI until the docs, cdn, and git repo disappear; and if that day arrive, then hopefully I'm in a position to help transfer the project to gh-pages.

### PS. Can you please confirm if the website is still down? and if not please close this issue. So that we can keep issues on topic.

**Comment by trsh** [comment]
@Morabaraba :/ . Website is UP, for now.

**Comment by Morabaraba** [comment]
Thanks @trsh 

If you want access to the api or manual and it seems that it is blocked you can also add `-cdn` to the subdomain eg.

- http://docs-cdn.qiciengine.com/manual/Sample/Node.html
- http://docs-cdn.qiciengine.com/api/

and if the domains expire you can access it at github:

- https://github.com/qiciengine/qiciengine-documentation
